selfdrive/controls/lib/radar_helpers.py

```diff
@@ -130,14 +130,19 @@ def get_RadarState(self, model_prob=0.0):
       "aLeadTau": float(self.aLeadTau)
     }
 
-  def get_RadarState_from_vision(self, lead_msg, v_ego):
+  def get_RadarState_from_vision(self, lead_msg, v_ego, vision_v_ego):
+    # Learn vision model velocity error to correct vlead
+    corrected_v_lead = lead_msg.v[0]
+    if v_ego > 0 and vision_v_ego > 0:
+      vision_velocity_factor = v_ego / vision_v_ego
+      corrected_v_lead = lead_msg.v[0] * vision_velocity_factor
```

Comment:
Might make more sense to copy what longitudinal_planner does, which is simply get an error to subtract

Reply:
should be a scale rather than a subtract, if there is an error, then it might apply to a static lead no?